server/routes/assessment.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+
+export default function createRouter(assessment) {
+  const router = express.Router();
+
+  router.post('/', (req, res) => {
+    assessment.record(req.body)
+      .then(
+        result => res.json({
+          status: 'OK',
+          data: { id: result.assessment_id },
+        }),
+        (err) => {
+          // TODO: proper logging lib for server stuff?
```

Comment:
Yeah, proper logging will be configured at the app, and then some middleware that also writes to the main logger per-request.